pkg/otel/templates/base.tmpl

```diff
@@ -0,0 +1,67 @@
+{{- define "aggregator-config" -}}
```

Comment:
Do we actually use this template anywhere?

Reply:
config validation tests for now

Reply:
will move these to pkg/test/testdata to be more idiomatic